packages/terra-show-hide/src/_ShowHideButton.jsx

```diff
@@ -135,18 +118,22 @@ class Button extends React.Component {
       <button
         {...customProps}
         className={buttonClasses}
-        type={type}
-        disabled={isDisabled}
-        tabIndex={isDisabled ? '-1' : undefined}
-        aria-disabled={isDisabled}
+        type="button"
         onKeyDown={this.handleKeyDown}
         onKeyUp={this.handleKeyUp}
         onBlur={this.handleOnBlur}
         onClick={onClick}
         onFocus={onFocus}
         ref={refCallback}
       >
-        {text}
+        <span className={cx('inner')}>
```

Comment:
Using flexbox to vertically center align icon when text wraps. Button can't be `display: flex` in some browsers so using inner spans to set up flexbox.